src/pages/basic_data/app_admin/CustomerList.tsx

```diff
@@ -0,0 +1,123 @@
+import EditIcon from '@mui/icons-material/Edit';
+import {
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { Box } from '@mui/system';
+import { visuallyHidden } from '@mui/utils';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link as RouterLink } from 'react-router-dom';
+import { alternatingTableRowColor } from '../../../themes/mainTheme';
+import { SimpleCustomerInstitution } from '../../../types/customerInstitution.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { getAdminInstitutionPath } from '../../../utils/urlPaths';
+import { SearchTextField } from '../../search/SearchTextField';
+
+interface CustomerListProps {
+  customerInstitutions: SimpleCustomerInstitution[];
+}
+
+enum CustomerStatusFilter {
+  ShowAll,
+  Active,
+  Inactive,
+}
+
+export const CustomerList = ({ customerInstitutions }: CustomerListProps) => {
+  const { t } = useTranslation();
+
+  const [customerStatus, setCustomerStatus] = useState<CustomerStatusFilter>(CustomerStatusFilter.ShowAll);
+  const [searchTerm, setSearchTerm] = useState('');
+  const statusFilteredCustomers =
+    customerStatus === CustomerStatusFilter.Active
+      ? customerInstitutions.filter((customer) => customer.active)
+      : customerStatus === CustomerStatusFilter.Inactive
+        ? customerInstitutions.filter((customer) => !customer.active)
+        : customerInstitutions;
+  const filteredCustomers = searchTerm
+    ? statusFilteredCustomers.filter((customer) =>
+        customer.displayName.toLowerCase().includes(searchTerm.toLowerCase())
+      )
+    : statusFilteredCustomers;
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+      <Box
+        sx={{ display: 'grid', gridTemplateColumns: '4fr 1fr', columnGap: '0.5rem', width: { lg: '100%', xl: '50%' } }}>
+        <SearchTextField
+          data-testid={dataTestId.basicData.customers.customerNameSearchField}
+          placeholder={t('basic_data.institutions.search_for_name')}
+          value={searchTerm}
+          onChange={(event) => setSearchTerm(event.target.value)}
+        />
+        <FormControl fullWidth>
```

Comment:
Mobilvisning er kanskje ikke verdens viktigste ting for dette da, men kanskje verdt å la brukeren se hva man velger her likevel siden det ikke bør være noe stor jobb?

![bilde](https://github.com/user-attachments/assets/1f10c0c2-cd5e-4fab-b7c8-ca0b230b4f2c)